gradle/libs.versions.toml

```diff
@@ -21,9 +26,8 @@ uiTextGoogleFonts = "1.6.8"
 androidx-core-ktx = { group = "androidx.core", name = "core-ktx", version.ref = "coreKtx" }
 androidx-navigation-compose = { module = "androidx.navigation:navigation-compose", version.ref = "navigationCompose" }
 androidx-ui-text-google-fonts = { module = "androidx.compose.ui:ui-text-google-fonts", version.ref = "uiTextGoogleFonts" }
-junit = { group = "junit", name = "junit", version.ref = "junit" }
+junit = { group = "junit", name = "junit", version = "4.13.2" }
```

Comment:
why don't you use `version.ref`?

Reply:
Also, why import a legacy version of JUnit when you already have JUnit 5 imported?

Reply:
beacuse build failed if I use version.ref, I am not sure why, so I just followed the suggestion by IDE to change the Requested Version to the latest version showed on dropdown list
![image](https://github.com/user-attachments/assets/398ec6c7-8347-4057-8f77-d77b1bbd0837)

Reply:
You had this error because the JUnit version specified by `version.ref` because you were trying to import JUnit 5 using JUnit 4's way.

Reply:
NVM, I fixed it